File: python/exercicio07.py
# Crie uma função em Python chamada 'filtrar_pares' que receba
# uma lista de números inteiros como argumento.
#
# A função deve retornar uma NOVA lista contendo APENAS os números pares da lista original.


def filtrar_pares(numeros):
    pares = []
    for numero in numeros:
        if numero % 2 == 0:
            pares.append(numero)
    return pares

entrada = input("Digite vários números separados por espaço: ")
numeros_do_usuario = [int(n) for n in entrada.split()]
numeros_pares_encontrados = filtrar_pares(numeros_do_usuario)
print("Números pares encontrados:", numeros_pares_encontrados)




            

File: javascript/exercicio07.js
//
// Crie uma função em JavaScript chamada 'filtrarNumerosPares' que receba
// um array de números inteiros como argumento.
//
// A função deve retornar um NOVO array contendo APENAS os números pares do array original.
//
// **Regras:**
// - O array original pode estar vazio.
// - Use um loop para percorrer o array.

function filtrarNumerosPares(numeros) {
  let pares = [];

  for (let i = 0; i < numeros.length; i++) {
    if (numeros[i] % 2 === 0) {
      pares.push(numeros[i]);
    }
  }

  return pares;
}

console.log(filtrarNumerosPares([1, 2, 3, 4, 5, 6, 77, 88]))